test/libgit2.jl

```diff
@@ -1910,8 +1921,11 @@ mktempdir() do dir
                             deserialize(f)
                         end
                         @test err.code == LibGit2.Error.ERROR
-                        @test err.msg == "Invalid Content-Type: text/plain"
+                        @test err.msg == "invalid Content-Type: text/plain"
```

Comment:
I think I'll update this and https://github.com/JuliaLang/julia/pull/22614#discussion_r124927761 to be case insensitive